core/modules/user/user.module

```diff
@@ -759,6 +767,20 @@ function user_account_form(&$form, &$form_state) {
   $form['#validate'][] = 'user_account_form_validate';
   module_load_include('password.inc', 'user', 'user');
   $reject_weak = user_password_reject_weak($user->name);
+
+  // Label the threshold for the password description.
+  $strength_threshold = $site_config->get('user_password_strength_threshold');
+  $threshold_label = t('excellent');
```

Comment:
I belief, this should be `strong` for consistency. This seems to be the only time the term `excellent` is used.

Reply:
"excellent" is used in the JS file. The end user will see "excellent", strong is just in the code.